assignments/callbacks.js

```diff
@@ -24,24 +24,57 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr)
 }
+function lengthOfArr(arr) {
+  return arr.length
+}
+console.log(getLength(items, lengthOfArr));
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr)
```

Comment:
you would want to pass the last item ot the array in this function, like this:
 cb(arr[arr.length - 1]);